server/src/graphql/resolvers/queries.js

```diff
@@ -31,4 +31,9 @@ module.exports = {
       return invalidUser;
     }
   },
+
+  getAllRigs: async(parent) => {
+    const allRigs = await db.Rig.findAll({ include: [db.Well] })
```

Comment:
I'd suggest that you can just `return` the await instead of making a variable only to return it.